react/features/shared-video/components/native/SharedVideoDialog.js

```diff
@@ -56,15 +56,15 @@ class SharedVideoDialog extends AbstractSharedVideoDialog<*> {
 
         return (
             <InputDialog
-                contentKey = 'dialog.shareVideoTitle'
                 messageKey = { error ? 'dialog.sharedVideoDialogError' : undefined }
                 onSubmit = { this._onSubmitValue }
                 textInputProps = {{
                     autoCapitalize: 'none',
                     autoCorrect: false,
                     placeholder: t('dialog.sharedVideoLinkPlaceholder'),
-                    placeholderTextColor: ColorPalette.lightGrey
-                }} />
+                    placeholderTextColor: BaseTheme.palette.text03
```

Comment:
I think we better leave this to the react-native-dialog package, it already knows what the right color is: https://github.com/mmazzarolo/react-native-dialog/blob/5a69064a1fe5c99f9866ddadddb7c197c2728acc/src/Input.tsx#L43